src/QsCompiler/DataStructures/SymbolTable.fs

```diff
@@ -538,6 +546,14 @@ and Namespace private
         let doc = ImmutableArray.Create(sprintf "automatically generated %A specialization for %s.%s" kind this.Name.Value parentName.Value)
         this.TryAddCallableSpecialization kind (source, location) ((parentName, declLocation.Range), generator, doc) 
 
+    /// Deletes the specialization(s) defined at the specified location and source file for the callable with the given name.
+    /// Throws an ArgumentException if the given source file is not listed as a source for (part of) the namespace.
+    /// Throws the standard key does not exist exception if no callable with that name exists.
+    member internal this.RemoveSpecialization (source, location) cName =
```

Comment:
Same comment from above about maybe adding a note describing the return value.

Reply:
Yup, will do.